doc/LedgerVersion.cmake

```diff
@@ -3,3 +3,4 @@ set(Ledger_VERSION_MINOR 3)
 set(Ledger_VERSION_PATCH 2)
 set(Ledger_VERSION_PRERELEASE "")
 set(Ledger_VERSION_DATE  20230330)
+set(Ledger_VERSION "${Ledger_VERSION_MAJOR}.${Ledger_VERSION_MINOR}.${Ledger_VERSION_PATCH}")
```

Comment:
This is for convenience, as much as I enjoy typing it get tedious at some point.